.travis.yml

```diff
@@ -29,6 +32,7 @@ matrix:
         - ssh default sudo podman run --privileged --cgroupns=private test make localunittest
   allow_failures:
     - go: tip
+    - name: "cgroup-v2"
```

Comment:
can we add retry code to the v2 setup script instead?

Reply:
Maybe, but the v2 tests also take the longest to run -- I'm not a huge fan of the idea of having possibly-regular >1hr test runs.

Reply:
> @cyphar: I'm not a huge fan of the idea of having possibly-regular >1hr test runs.

Compared to how little the CI is exercised here, that seems like an acceptable trade-off for not breaking things...? "Acceptable failures" often quickly degenerate unless someone pays close attention to them. Wouldn't it be better to try to figure out what's causing the flakes and fixing that?